otter/constants.py

```diff
@@ -52,10 +59,22 @@ def get_service_configs(config):
         configs[ServiceType.CLOUD_METRICS_INGEST] = {
             'name': metrics['service'], 'region': metrics['region']}
 
+    # {"cloudfeeds"...} contains config for 2 feeds services: One where otter
+    # pushes scaling group events `otter.log.cloudfeeds` as {"url": ...}
+    # and other where it fetches customer access events for terminator which
+    # is {"customer_access_events": {"url": "http://url"} or
```

Comment:
A always B be C cryptographically verifying even hypothetical network connectivity